src/services/CardApi.js

```diff
@@ -195,4 +195,19 @@ export class CardApi {
 			})
 	}
 
+	findAllWithDue(data) {
+		return axios.get(this.url(`/cards/findAllWithDue`))
+			.then(
+				(response) => {
+					return Promise.resolve(response.data)
+				},
```

Comment:
```suggestion
				(response) => Promise.resolve(response.data),
```

Same for the other arrow functions :wink: